test-apps/imodel-transformer/README.md

```diff
@@ -6,8 +6,40 @@ The application contained within this directory is a test application for transf
 
 Example command line:
 
-`npm start -- --source=<sourceIModelFileName> --target=<targetIModelFileName>`
+`npm start -- --sourceFile=<sourceIModelFileName> --targetFile=<targetIModelFileName>`
 
 To get usage help run:
 
 `npm start -- --help`
+
+## imodel-transformer as a sample application
+
+This application demonstrates the following:
+
+- Using the `IModelTransformer` API
+- Calling the `ContextRegistry` API to get *context* (project or asset) information
+- Querying `IModelHub` to get iModel information
+- How to handle either *briefcase* or *snapshot* iModel in the same application
+- Using `yargs` to handle command line arguments

```

Comment:
perhaps a link `[yargs](http://yargs.js.org/)` would be useful here? Same goes for some of these other references which could reference to the latest docs site:
- IModelHub
- IModelTransformer
- ContextRegistry
- Context, Briefcase, Snapshot

Reply:
I am going to take your suggestion for `yargs`, but leave the others alone. Given that this is in the `imodeljs` repository, I am going to assume that people can find those classes/APIs. If this test-app were in a separate repository, then I would definitely want the links that you suggested.

Reply:
resolving but I have to admit, I find it helpful when documentaiton is strongly linked to save me from manually searching for it.